src/platform/silabs/multi-ota/OTAMultiImageProcessorImpl.cpp

```diff
@@ -419,10 +426,14 @@ void OTAMultiImageProcessorImpl::HandleApply(intptr_t context)
 
     imageProcessor->mAccumulator.Clear();
 
-    ChipLogProgress(SoftwareUpdate, "HandleApply: Finished");
+    ChipLogProgress(SoftwareUpdate, "HandleApply: Finished and Soft Reset initiated");
```

Comment:
nit: This print will never appear. Device reboot before it has the chance to print. A small delay was added in the default efr32 ota processor

Reply:
Ok I have added delay before `CORE_CRITICAL_SECTION(bootloader_rebootAndInstall();`
Looks like many changes need to be adopted to multi-ota from default ota implementation